From fe704ae0f6af08e7f3e869006ea52c0e03466d14 Mon Sep 17 00:00:00 2001
Message-ID: <fe704ae0f6af08e7f3e869006ea52c0e03466d14.1736059920.git.divya@subvertising.org>
In-Reply-To: <cover.1736059918.git.divya@subvertising.org>
References: <cover.1736059918.git.divya@subvertising.org>
From: Divya Ranjan <divya@subvertising.org>
Date: Sat, 4 Jan 2025 21:43:14 +0000
Subject: [PATCH 42/44] gnu: Add rust-nu-utils-0.88.

* gnu/packages/crates-io.scm (rust-nu-utils-0.88): New variable.
---
 gnu/packages/crates-io.scm | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/gnu/packages/crates-io.scm b/gnu/packages/crates-io.scm
index e34f470f8c..5c1a16a826 100644
--- a/gnu/packages/crates-io.scm
+++ b/gnu/packages/crates-io.scm
@@ -47681,6 +47681,31 @@ (define-public rust-nu-test-support-0.88
     (description "This package provides Support for writing Nushell tests.")
     (license license:expat)))
 
+(define-public rust-nu-utils-0.88
+  (package
+    (name "rust-nu-utils")
+    (version "0.88.1")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (crate-uri "nu-utils" version))
+       (file-name (string-append name "-" version ".tar.gz"))
+       (sha256
+        (base32 "1g84zbs7c4yyc3x1wbml683amh13y9v917ags6y5jv3zvn0yv7ik"))))
+    (build-system cargo-build-system)
+    (arguments
+     `(#:cargo-inputs (("rust-crossterm-winapi" ,rust-crossterm-winapi-0.9)
+                       ("rust-log" ,rust-log-0.4)
+                       ("rust-lscolors" ,rust-lscolors-0.15)
+                       ("rust-num-format" ,rust-num-format-0.4)
+                       ("rust-strip-ansi-escapes" ,rust-strip-ansi-escapes-0.2)
+                       ("rust-sys-locale" ,rust-sys-locale-0.3)
+                       ("rust-unicase" ,rust-unicase-2))))
+    (home-page "https://github.com/nushell/nushell/tree/main/crates/nu-utils")
+    (synopsis "Nushell utility functions")
+    (description "This package provides Nushell utility functions.")
+    (license license:expat)))
+
 (define-public rust-nucleo-0.2
   (package
     (name "rust-nucleo")
-- 
2.47.1

